File: Lógica de Programação Java e C#/5 - Arrays/Java/Exercícios de Fixação/SequenciaAleatoria.java
class SequenciaAleatoria{
	public static void main(String [] args){
		int [] vetor = new int [10];
			for(int a=0;a<vetor.length;a++){				
				vetor[a]=(int)(Math.random()*10);
				System.out.println(vetor[a]);
			}
	}
}
			

File: Lógica de Programação Java e C#/5 - Arrays/C#/Exercício de Fixação/SequenciaAleatoria.cs
class SequenciaAleatoria
{
    static void Main()
    {
        int[] vetor = new int[10];
        System.Random gerador = new System.Random();
        for (int a = 0; a < vetor.Length; a++)
        {
            vetor[a] = (int)(gerador.NextDouble()*10);
            System.Console.WriteLine(vetor[a]);
        }
    }
}